src/utils/SelectionValidation.js

```diff
@@ -0,0 +1,37 @@
+export class SelectionValidation {
+    constructor(){
+        this.status = null;
+        this.validationData = null;
+    }
+//    debounceValidation(func, timeout){
+//    console.log('debounceValidation')
+//        let timer;
+//          return (...args) => {
+//            clearTimeout(timer);
+//            timer = setTimeout(() => { func.apply(this, args); }, timeout);
+//          };
+//        }
```

Comment:
Delete

Reply:
Currently leaving because debouncing could potentially reduce input lag. Something about it broke, so we commented it out for the time being. Input lag isn't a problem currently, but felt wrong to get rid of! Open to thoughts